SMLHelper/Assets/PdaItem.cs

```diff
@@ -9,7 +9,6 @@
     /// <seealso cref="Spawnable" />
     public abstract class PdaItem : Spawnable
```

Comment:
Not entirely related to this PR per-say, but mark this as `[Obsolete]` so people don't use it

Reply:
I would remove that warning line from the documentation instead.

Anyone who finds use for this class is welcome to use it, so long as they are careful about it.